DEV_GUIDE.md

```diff
@@ -26,6 +27,22 @@ This document gives a detailed breakdown of the various build processes and opti
   * [Rendering documentation](#rendering-documentation)
 <!-- TOC -->
 
+## Project structure
+
+The module structure is as follows:
+
+* `kroxylicious-bom` the Bill Of Materials used by developers to conveniently declare a consistent set of Kroxylicious dependencies.
+* `krpc-code-gen` a code generator used to build the API module.
+* `kroxylicious-api` the APIs needed to write a filter.
+* `kroxylicious-unit-test` test utilities for filter authors.
+* `kroxylicious` the proxy runtime platform for hosting filters, including the networking code etc, but lacking a `main()` method
+* `kroxylicious-test-tools` ???
+* `kroxylicious-app` provides a `main()` method for the proxy.
+* `kroxylicious-multitenant` a filter that implements multitenancy
+* `kroxylicious-schema-enforcement` a filter that implements schema enforcement
+* `integrationtests` the integration test suite
```

Comment:
Every other module uses `-` as a word separate, but not this one. I think we should rename it. I also don't know why it lacks the `kroxylicious` prefix.